terraform/cloud_function/pubsub_triggered/variables.tf

```diff
@@ -0,0 +1,17 @@
+variable "bucket_name" {}
+variable "branch_suffix" {}
+variable "entry_point" {
+  default = "handler"
+}
+variable "function_env_vars" {}
```

Comment:
This needs a default variable since we don't always need to pass environment vars